jupyter_schemas.py

```diff
@@ -59,3 +61,61 @@ def list_schemas(project: typing.Optional[str] = None) -> typing.List[str]:
             logging.warning(f"Could not find an ID/URI in {fpath}.")
             pass
     return schema_uris
+
+
+class Schema:
```

Comment:
I wrote a _very_ similar class to this in [Jupyter Events here](https://github.com/jupyter/jupyter_events/blob/main/jupyter_events/schema.py).

I wonder if we can consolidate some of this logic.